submissions/BlessingOkoro/html/AboutUs.html

```diff
@@ -4,30 +4,86 @@
+
+            <div class="divide1">
+                <h3>Buy in bulk for party planning</h3>
+                <p class="second-paragraph">
+                    Whether planning dinner or a party, we have all the necessities, including main dishes, spices, meal sides, and desserts, to help you pull it off with flavorful zest! We offer both frozen and fresh products to accommodate a range of needs.
+                </p>
+            </div>
+
+
+            <div class="divide1">
+                <h3>Avoid the hassle of the grocery store</h3>
+                <p class="second-paragraph">
+                    Choose from popular options such as T-bone steaks, shrimps, chicken wings, and pork chops. At <em>Fresh & Frozen Company </em> we are out to expand our reach to include the entire country with our fast, efficient delivery services. You no longer need a grocery store,we got you covered.
+                </p>
+            </div>
+    </div>
+</div>
+
+ 
+<!-------         <p>With so many benefits, the decision to buy meats and seafood online makes solid sense! When choosing to buy seafood online, we are a top choice for many loyal customers. At <em>Fresh & Frozen Company </em> we are out to expand our reach to include the entire country with our fast, efficient delivery services. We offer one of the widest selections of products, including various seafood, beef, pork, sausage, and other meats, to make meal planning a breeze. We also proudly feature an array of specialty meats and gourmet products, including rabbit, turtle, frog, shark and more, to help expand your palate. No matter what type of meat or seafood you need, we have you covered. Choose from popular options such as T-bone steaks, shrimps, chicken wings, and pork chops. Whether planning dinner or a party, we have all the necessities, including main dishes, spices, meal sides, and desserts, to help you pull it off with flavorful zest! We offer both frozen and fresh products to accommodate a range of needs. <br>
```

Comment:
@bnokoro You can remove this comment if you don't intend on using it. You should avoid leaving comments in your code as much as possible

Reply:
@Gbensky , thanks for the  review,  i  have removed the comments in my code, and i made a commit towards the change. I noticed that in the files changed here on git hub, the comments are still there, how do i remove that too because i no longer have the files on my pc, thanks

Reply:
I will approve this PR but once it is merged, please pull your master branch and remove the comment